src/test/client/api.vscode.test.ts

```diff
@@ -24,13 +24,13 @@ import { executeSilently } from '../../kernels/helpers';
 import { getPlainTextOrStreamOutput } from '../../kernels/kernel';
 import { IInterpreterService } from '../../platform/interpreter/contracts';
 
-suite('3rd Party Kernel Service API', function () {
+suite.only('3rd Party Kernel Service API', function () {
```

Comment:
Got a .only in here. The PR into non main makes it a bit hard to catch these since the linting step doesn't run.